### PR TITLE
Update min velocity in grow_workflow

### PR DESCRIPTION
**Pull request by KobusVanRooyen**
Updated the min velocity setting to 1e-4 which is should cater for a DN800 connected to a 10kW heat demand

- [x] Remove the min velo = 0.0
- [x] Create a test to catch this in the future
- [x] Ensure the default value exists and is used
- [x] Pipeline test pass
- [x] Pipeline formatting pass

### CODE REVIEW

**Comment by KobusVanRooyen** [comment]
@FJanssen-TNO ready for review

**Comment by KobusVanRooyen** [comment]
@FJanssen-TNO ready for review

File: src/mesido/workflows/grow_workflow.py [hunk 1]
```diff
@@ -133,7 +133,8 @@ class EndScenarioSizing(
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
-        self.heat_network_settings["minimum_velocity"] = 0.0  # 0.001
+        self.heat_network_settings["minimum_velocity"] = 1.0e-4
```
Comment (FJanssen-TNO) [inline src/mesido/workflows/grow_workflow.py:136]: Add a comment why this value is picked, above this line of code.

File: tests/test_updated_esdl_pre_process.py [hunk 2]
```diff
@@ -48,6 +59,10 @@ def test_updated_esdl(self):
         file.write(optimized_esdl_string)
         file.close()
 
+        demand_matching_test(problem, problem.extract_results())
+        energy_conservation_test(problem, problem.extract_results())
+        heat_to_discharge_test(problem, problem.extract_results())
+
```
[the remark this answers is not in the document]
Comment (KobusVanRooyen) [reply tests/test_updated_esdl_pre_process.py:65]: Yes they were missing

File: src/mesido/workflows/grow_workflow.py [hunk 3]
```diff
@@ -231,7 +232,8 @@ def energy_system_options(self):
         #  constraints in the ESDL e.g. min max pressure
         options = super().energy_system_options()
         options["maximum_temperature_der"] = np.inf
-        options["heat_loss_disconnected_pipe"] = True
+        # options["heat_loss_disconnected_pipe"] = True
+        options["heat_loss_disconnected_pipe"] = False
```
[the remark this answers is not in the document]
Comment (KobusVanRooyen) [reply src/mesido/workflows/grow_workflow.py:236]: My mistake. It has been corrected now

File: tests/test_head_loss.py [hunk 4]
```diff
@@ -244,33 +244,59 @@ def test_heat_network_pipe_split_head_loss(self):
...
+                    elif (
+                        head_loss_option_setting
+                        == HeadLossOption.LINEARIZED_N_LINES_WEAK_INEQUALITY
+                    ):
                         self.heat_network_settings["minimize_head_losses"] = True
+                        if counter_linearized_n_lines_weak_ineq_runs == 1:
+                            self.heat_network_settings["minimum_velocity"] = 0.0
+                        elif counter_linearized_n_lines_weak_ineq_runs == 2:
+                            ...
+                            # with min velo = default value (>0.0), instead of specifying a value
```
[the remark this answers is not in the document]
Comment (KobusVanRooyen) [reply tests/test_head_loss.py:298]: No extra code needed. The point here is to remind the dev that the min velo is not zero when the counter_linearized_n_lines_weak_ineq_runs == 2. I have added extar comment. This should stay

File: src/mesido/workflows/grow_workflow.py (hunk 1 above)
Comment (KobusVanRooyen) [reply src/mesido/workflows/grow_workflow.py:136]: Nice for picking this up. I added a comment

**Review by FJanssen-TNO — APPROVED** [review]

**Pull request merged**